cicd/jenkins/important-changes-to-openshift-jenkins-images.adoc

```diff
@@ -19,7 +19,7 @@ include::modules/relocation-of-openshift-jenkins-images.adoc[leveloffset=+1]
 include::modules/customizing-the-jenkins-image-stream-tag.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
-[id="additional-resources_openshift-jenkins_{context}"]
+[id="additional-resources_important-changes-to-openshift-jenkins-images"]
```

Comment:
```suggestion
[id="additional-resources_important-changes-to-openshift-jenkins-images_{context}"]
```
I think `{context}` is required here.

Reply:
I wondered about that, too. Because the assembly is always its own unique context, this seems redundant. I did a survey of other assembly files in the repo and some recent PRs: it's no guarantee, but I didn't see them using `_{context}` either.

Reply:
No harm in adding it, either. Done.